### PR TITLE
Time stamperror

### PR DESCRIPTION
**Pull request by LiadAzulay**

### CODE REVIEW

File: projects/angular-proto-firestore/src/app/app.component.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+<div *ngIf="!storylist">
+    Loading
 </div>
+<div *ngIf="storylist">
+    <div *ngFor="let storylistChild of storylist">
+        ----
+        <div *ngFor="let story of storylistChild.getStoryList()">
+            {{ story.getProject()}} :
+            <div *ngFor="let storyItem of story.getItemList()">
+                <!-- doesn't show anything -->
```
Comment (webdevelopland) [inline projects/angular-proto-firestore/src/app/app.component.html]: It's ok, there's no data inside:
https://i.imgur.com/FwMcB69.jpg

File: projects/angular-proto-firestore/src/app/core/services/firebase.service.ts [hunk 2]
```diff
@@ -3,47 +3,54 @@ import { AngularFirestore, AngularFirestoreCollection } from 'angularfire2/fires
...
-  getBook(): Observable<Book> {
-    return this.protobin
-      .doc('book')
-      .snapshotChanges()
-      .pipe(
-        map(action => {
-          const firebaseElement = action.payload.data() as FirebaseElement;
+  getstorylistAll(): Observable<StoryList[]> {
+    return this.protobin.snapshotChanges().pipe(
+        map(action => action.map(a => {
```
Comment (webdevelopland) [inline projects/angular-proto-firestore/src/app/core/services/firebase.service.ts]: :+1:

File: projects/angular-proto-firestore/src/app/import/firebase/firebase.module.ts [hunk 3]
```diff
@@ -2,24 +2,31 @@ import { NgModule } from '@angular/core';
...
 @NgModule({
   imports: [
     AngularFireModule.initializeApp(config),
     AngularFirestoreModule,
     AngularFireAuthModule,
+    
+    
   ],
   declarations: [],
   providers: [
```
Comment (webdevelopland) [inline projects/angular-proto-firestore/src/app/import/firebase/firebase.module.ts]: Here!
```
providers: [
  AngularFireDatabase,
  AngularFireAuth,
  { provide: FirestoreSettingsToken, useValue: {} }
]
```

File: projects/angular-proto-firestore/src/app/app.component.html (hunk 1 above)
Comment (oferb) [reply projects/angular-proto-firestore/src/app/app.component.html]: Right, right now the desktop client is not uploading any images.
We need to add that..

**Pull request closed without merging**